charts/kubefed/templates/federatedtypeconfig.yaml

```diff
@@ -58,7 +58,7 @@ spec:
 apiVersion: core.kubefed.io/v1beta1
 kind: FederatedTypeConfig
 metadata:
-  name: ingresses.extensions
```

Comment:
`extensions/v1beta1.Ingress` was deprecated in k8s v1.14 and `networking.k8s.io/v1.Ingress` is available from k8s v1.19 so kubefed will still support ingresses back to v1.19 - this is an acceptable support window considering v1.24 is now out.

Reply:
We should make sure to properly communicate this breaking change.

Reply:
I'm not sure if this is a breaking change :thinking: Anyone using a newer version of Kubernetes will not be able to propagate ingress without this change... the breaking change is in the deletion of the API in Kubernetes, this is actually a fix!

Reply:
Pulled this change out into #1521 to keep the PRs small and focused.

Reply:
I tend to agree with @jimmidyson. This is not a breaking change, but may be worth documenting anyways. @jimmidyson you also mentioned that you will cut a new release post this. It should ideally be sufficient to explicitly point this out in the release notes.

Reply:
@irfanurrehman I'll make sure to do that.